src/fxos/index.js

```diff
@@ -0,0 +1,13 @@
+let modularize = require('../modularize');
+
+let Fxos = modularize(function Fxos(adb) {
```

Comment:
How would you be able to specify which Adb instance to use for Fxos if Fxos wasn't also request-specific?